app/src/main/java/com/github/muellerma/mute_reminder/MediaAudioManager.kt

```diff
@@ -9,11 +9,14 @@ import androidx.core.content.getSystemService
 
 class MediaAudioManager(context: Context) {
     private val audioManager: AudioManager = context.getSystemService()!!
+    private val prefs = Prefs(context)
 
     fun shouldNotify(): Boolean {
-        return isRingToneMuted() && !isMediaMuted() && !usesRemoteOutput()
+        return (isRingToneMuted() || isRingToneIgnored()) && !isMediaMuted() && !usesRemoteOutput()
     }
 
+    private fun isRingToneIgnored() = prefs.ignoreRingTone
```

Comment:
I'd drop this function.

Reply:
I thought it was cleaner to do like this as all the condition are functions, I can remove it tho if you prefer